packages/babel-traverse/src/scope/index.js

```diff
@@ -952,11 +951,11 @@ export default class Scope {
    */
 
   getAllBindings(): Object {
-    const ids = Object.create(null);
+    let ids = Object.create(null);
```

Comment:
```suggestion
    const ids = Object.create(null);
```

Once the other suggested change is made.